src/app/cli/src/services/account_service.rs

```diff
@@ -101,8 +252,25 @@ impl CurrentAccountIndication {
     }
 
     pub fn as_current_account_subject(&self) -> CurrentAccountSubject {
```

Comment:
Nitpick: In rust:
- `as_*()` usually means a zero overhead type conversion (e.g. `String` to `&str`)
- `into_*()` usually means conversion that consumes `self` and returns converted type
- `to_*()` creates a new instance of a target type while leaving original intact (e.g. `Path::to_path_buf()`)

Reply:
`to_*()` is the right match here